bin/yaml/racket.yaml

```diff
@@ -0,0 +1,13 @@
+compilers:
+  racket:
+    type: script
+    check_exe: bin/racket --version
+    dir: racket-{name}
+    fetch:
+      - https://download.racket-lang.org/releases/{name}/installers/racket-{name}-x86_64-linux-cs.sh racket-{name}.sh
+    script: |
+      sh racket-{name}.sh --in-place --dest ./racket-{name}
+    after_stage_script:
+      - bin/raco pkg install -i --auto disassemble
```

Comment:
And this? Does it also keep everything in the final compiler folder?

Reply:
Yes, this is also self-contained. The `-i` argument means "installation scope", which equates to the `share/pkgs` subdirectory of the compiler folder.